src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/RazorLSPTextViewConnectionListener.cs

```diff
@@ -63,11 +109,81 @@ public void SubjectBuffersConnected(ITextView textView, ConnectionReason reason,
             }
 
             RazorLSPTextViewFilter.CreateAndRegister(vsTextView);
+
+            // Initialize the user's options and start listening for changes.
+            // We only want to attach the option changed event once so we don't receive multiple
+            // notifications if there is more than one TextView active.
+            if (!textView.TextBuffer.Properties.ContainsProperty(RazorEditorOptions) &&
```

Comment:
I tried using `typeof(IEditorOptions)` here, but it was already in use so I settled on creating the `RazorEditorOptions` const.

Reply:
This seems fine